src/components/AxisLabel/AxisLabel.spec.jsx

```diff
@@ -102,7 +107,11 @@ describe('AxisLabel', () => {
 		describe('label', () => {
```

Comment:
That's a good point -- as part of this PR I didn't go through and convert any existing tests over to using snapshots, or remove tests that are replaced with Example snapshots.

I'll make an effort to do some of that now, but it's not likely that I'm going to catch everything.